pkg/operator/ceohelpers/bootstrap.go

```diff
@@ -146,12 +146,12 @@ func CheckSafeToScaleCluster(
 		return fmt.Errorf("CheckSafeToScaleCluster %d nodes are required, but only %d are available", minimumNodes, nodeCount)
 	}
 
-	healthyMembers, err := etcdClient.HealthyMembers(context.Background())
+	memberHealth, err := etcdClient.MemberHealth(context.Background())
```

Comment:
@tjungblu the tests were failing because the `etcdfakeclient` was returning always empty slice for memberhealth :/ 

i tried many fixes, but this is the cleanest, ptal